omniscidb/CMakeLists.txt

```diff
@@ -309,19 +309,8 @@ find_package(ZLIB REQUIRED)
 
 option(ENABLE_FOLLY "Use Folly" ON)
 if(ENABLE_FOLLY)
-  find_package(Folly)
-  if(NOT Folly_FOUND)
-    set(ENABLE_FOLLY OFF CACHE BOOL "Use Folly" FORCE)
-  else()
-    include_directories(${Folly_INCLUDE_DIRS})
-    add_definitions("-DHAVE_FOLLY")
-    list(APPEND Folly_LIBRARIES ${Glog_LIBRARIES})
-    # TODO: use Folly::folly_deps?
-    if(MSVC)
```

Comment:
Are you sure this fix for Windows is not required anymore?

Reply:
They link it in the dependencies https://github.com/facebook/folly/blob/main/CMake/folly-deps.cmake#L70
I also saw libevent in the list of linked libraries in `folly-targets.cmake` that's installed.
I'm not 100% sure, but should work.

Reply:
Sounds good. Thanks!